assets/package.json

```diff
@@ -4,19 +4,19 @@
   "private": true,
   "engine-strict": true,
   "engines": {
-    "node": "^14.18",
-    "npm": "^6.14"
+    "node": "^16.13",
+    "npm": "^8.1"
   },
   "dependencies": {
     "simple-css-reset": "^3.0.0"
   },
   "devDependencies": {
     "babel-eslint": "^10.1.0",
```

Comment:
You can switch this to `@babel/eslint-parser` to fix the CI :)

Reply:
I don’t understand why it is not pulling the dep from the plugin since it’s defined there anyway!?

https://github.com/mirego/eslint-plugin-mirego/blob/master/package.json#L18

Reply:
It's listed as a devDependency, I'll go ahead and fix that..